psychopy/experiment/components/sound/__init__.py

```diff
@@ -77,6 +78,12 @@ def __init__(self,
             updates='constant',
             hint=msg,
             label=_translate("Sync start with screen"))
+        # --- Playback params ---
+        self.order += [
```

Comment:
`self.order` isn't created concisely with all the correct parameters. We should fix it. See my PR to your branch.

Reply:
Generally I prefer the approach of doing `+=` for each section, as it just makes it less of a monolith to maintain. e.g. if I add one param to Playback, I only have to think of it in relation to the params on the same page as it, rather than a block of all params at the top. This is just stylistic preference though, works the same either way :)